Gems/Atom/Feature/Common/Assets/Passes/DownsampleSinglePassMipChain.pass

```diff
@@ -0,0 +1,30 @@
+{
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "PassAsset",
+    "ClassData": {
+        "PassTemplate": {
+            "Name": "DownsampleSinglePassMipChainTemplate",
+            "PassClass": "DownsampleSinglePassMipChainPass",
+            "PassData": {
+                "$type": "ComputePassData",
+                "Make Fullscreen Pass": false,
```

Comment:
Is this a recognized key?

Reply:
DownsampleSinglePassMipChainPass is a subclass of ComputePass and its PassData should be given as ComputePassData.  The value of the key "Make Fullscreen Pass" has its default value as false, so I think I can omit this PassData description if it is desired.